_posts/2016-03-17-the-need-for-fit.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "The need for fit"
+date:   2016-03-17 11:58:26
+author: Michael Peyton Jones
+categories:
+---
+
+Personal fit with project the is incredibly important in startup founders. 
+Startup work is demanding: it typically involves working long hours for several years,
+with the prospect of a high failure rate even if you do keep going.
+This can be understandably [demoralizing](http://www.paulgraham.com/die.html),
+so a good personal fit with the project improves your chances of success both by
+making you more effective at the object level work, and by preventing failure
+due to loss of motivation[^others].
+
+[^others]: See [other](https://80000hours.org/2014/10/interview-holden-karnofsky-on-the-importance-of-personal-fit/) [people](https://80000hours.org/articles/personal-fit/) talking about personal fit.
+
+However, this thows some doubt on our strategy here at GTP. Our current plan is
```

Comment:
Most of this article is a general discussion of how fit might impact GTP's strategy. I think much better would be an analysis of the evidence for the importance of fit in entrepreneurship and how we think altruistic entrepreneurs should handle it.